addon/components/bsy-input.js

```diff
@@ -1,42 +1,43 @@
+import Ember from 'ember';
 import Component from '@ember/component';
 import { computed } from '@ember/object';
 import layout from '../templates/components/bsy-input';
 
+const { generateGuid } = Ember;
+
 const BsyInputComponent = Component.extend({
   layout,
-  tagName: "div",
-  classNames: "form-group",
-  type: "text",
+  tagName: 'div',
+  classNames: 'form-group',
+  type: 'text',
   readonly: false,
   placeholder: '',
   disabled: false,
-  classNameBindings: ["value:filled", "icon:has-icon", "formGroupSize"],
+  classNameBindings: ['value:filled', 'icon:has-icon', 'formGroupSize'],
   size: null, // sm, lg
 
-  inputClassNames: computed('size', function () {
+  inputClassNames: computed('size', function() {
     let size = this.get('size');
 
     if (size) {
       return `form-control input-${size}`;
     }
 
-    return "form-control";
+    return 'form-control';
   }),
 
-  iconSize: computed('size', function () {
-    let size = this.get('size');
-
-    if (size) {
-      return `icon--${size}`;
-    }
+  iconSize: computed('size', function() {
+    let { size } = this;
```

Comment:
I believe in these cases we don't define a variable but just refer to it via `this`. Same in `formGroupSize`.

Reply:
Not a rule but sure let's have it like that.